optimum/exporters/openvino/__main__.py

```diff
@@ -332,6 +333,48 @@ class StoreAttr(object):
                 return model
 
             GPTQQuantizer.post_init_model = post_init_model
+    elif library_name == "diffusers" and is_safetensors_available() and is_openvino_version(">=", "2024.6"):
+        if Path(model_name_or_path).is_dir():
```

Comment:
Can you please extract and encapsulate this code to a function with a meaningful name?

Reply:
done